src/Collapse.jsx

```diff
@@ -11,7 +12,8 @@ function toArray(activeKey) {
   return currentActiveKey;
 }
 
-const Collapse = React.createClass({
+const createReactClass = require('create-react-class');
```

Comment:
Thank you for your comment.
I changed the `createReactClass` code to class style component.
Please review.